QMLComponents/utilities/qutils.cpp

```diff
@@ -353,6 +354,11 @@ bool QColumnUtils::isDoubleValue(const QString &value)
 	return ColumnUtils::isDoubleValue(fq(value));
 }
 
+void QColumnUtils::setNumberOptionsOnQLocale(QLocale & locale)
+{
+	locale.setNumberOptions(QLocale::OmitGroupSeparator);
```

Comment:
Now can we get rid of the thousands separator by default if we add this in the settings?

Reply:
We could also add a setting for removing it completely yes.
But this seems like overkill?

Reply:
im adding it